charts/atlantis/templates/statefulset.yaml

```diff
@@ -220,7 +220,7 @@ spec:
             value: {{ .Values.atlantisUrl }}
           {{- else if .Values.ingress.enabled }}
           - name: ATLANTIS_ATLANTIS_URL
-            value: http://{{ .Values.ingress.host }}
+            value: https://{{ .Values.ingress.host }}
```

Comment:
this is a backwards incompatible change, I'm not sure it makes sense to do this. This can still be set to https if you want, correct?

Reply:
I'm not sure how you define "backwards incompatible change".

If someone is setting the `tls` flag in their ingress, their ingress is probably redirecting the `http` traffic to `https` (certainly ours does), which means that the client pretty much already has to support both `https` and following redirects. Users aren't going to run clients that don't.

Can you provide an explanation of how you'd imagine a user would configure their system to not want this behavior?

Reply:
guarding by tls makes sense